content/author.yaml

```diff
@@ -248,3 +248,7 @@
   bio: A seasoned IT professional with over 2.5 decades of IT experience in Coding, Design, Networking, Database Administration, Customer Support, Training, QA and consulting.
   linkedin: "sudhanshu-pandey-63b02a11" 
 
+- id: Anup Aglawe
+  bio: A computer science and engineering undergraduate student at IIT Gandhinagar. I immensly interested in developing applications and utlities.
```

Comment:
Please fix the grammatical issue in `I immensly interested in developing applications and utlities.`. It should be `I am immensely interested in developing applications and utilities.`